docs/actionable-messages/card-reference.md

```diff
@@ -63,6 +63,7 @@ If your card represents multiple "entities" or is, for instance, a digest for a
 |-------|------|-------------|
 | `@type` | String | Required. Must be set to `MessageCard`. |
 | `@context` | String | Required. Must be set to `http://schema.org/extensions`. |
+| `correlationId` | UUID | Set unique UUID for troubleshooting purposes.<br><br>**Recommended** to log before sending email or connector message.<br>For actions troubleshooting, **recommended** to log request header 'Card-Correlation-Id' and 'Action-Request-Id' in action execution request to your service endpoint. |
```

Comment:
Why do you have a `<br>` tag?

Reply:
Markdown tables are limited in what you can do inside a cell. In order to have line breaks, you have to fallback to HTML markup. This is because the table parsing breaks if you don't have a row all on one line :).